static/js/components/subject_page/main_pane.tsx

```diff
@@ -37,11 +42,11 @@ interface MainPanePropType {
   /**
    * Config of the page
    */
-  pageConfig: TopicPageConfig;
+  pageConfig: SubjectPageConfig;
   /**
-   * Summary for all topic page configs
+   * Summary of all available page configs
    */
-  topicsSummary: TopicsSummary;
+  allConfigsSummary: SubjectPageConfigSummary;
```

Comment:
Updated it so that instead of one SubjectPage component, there are SubjectPageMainPane and SubjectPageSidebar  components that users of the SubjectPage config can use to create their page. This should allow for more variation in the pages that use SubjectPage config.